ruby.c

```diff
@@ -1054,6 +1054,17 @@ setup_yjit_options(const char *s, struct rb_yjit_options *yjit_opt)
     else if (opt_match_noarg(s, l, "test-backend")) {
         yjit_opt->test_backend = true;
     }
+#if YJIT_JITDUMP
```

Comment:
May want to add that under the --yjit arguments?

Reply:
I think this is, `--yjit-jitdump`. The macro is to test that it's available (ie. we are on linux)